### ISSUE TITLE
Multiple errors on Amass, rgf , portscan, masscan , gowitness, etc while running scan.

### ISSUE THREAD

**Issue by Aijaz2005** [j3ssie/osmedeus#189]
Hi,

First of all thanks for the wonderful tool which you have created I very impressive and really liked the web UI part

, however when I installed and ran it on my VPS server with OS running Debian 9 on 10$ digital ocean machine. I encountered multiple issues when I ran the command :  

`./osmedeus.py -t hackerone.com -m subdomain,portscan,vuln --report full
`

```
[*] Execute: /root/Osmedeus/plugins/go/amass enum -timeout 10 -active -max-dns-queries 10000 -include-unresolvable -dir /root/.osmedeus/workspaces/hackerone.com/subdomain/amass-hackerone.com -d hackerone.com -o /root/.osmedeus/workspaces/hackerone.com/subdomain/hackerone.com-amass.txt
flag provided but not defined: -include-unresolvable


[*] Execute: /root/Osmedeus/plugins/go/rgf -dir /root/.osmedeus/workspaces/hackerone.com/fingerprint/responses/ | tee /root/.osmedeus/workspaces/hackerone.com/fingerprint/rgf-all-hackerone.com.txt
/bin/sh: 1: /root/Osmedeus/plugins/go/rgf: not found


[*] Execute: /root/Osmedeus/lib/alias/portscan -i hackerone.com -o '/root/.osmedeus/workspaces/hackerone.com/portscan/hackerone.com' -s '/root/.osmedeus/workspaces/hackerone.com/portscan/summary.txt' -p '/root/Osmedeus/plugins'
FAIL: unknown command-line parameter "hackerone.com"
 [hint] did you want "--hackerone.com"?
[*] Execute: sudo masscan --rate 10000 -p0-65535 hackerone.com -oX /root/.osmedeus/workspaces/hackerone.com/portscan/hackerone.com.xml --wait 0
[-] No open port found


[*] Execute: /root/Osmedeus/plugins/go/gowitness file -s /root/.osmedeus/workspaces/hackerone.com/vulnscan/scheme-hackerone.com.txt -t 30 --log-level fatal --destination  /root/.osmedeus/workspaces/hackerone.com/vulnscan/screenshot/raw-gowitness/ --db /root/.osmedeus/workspaces/hackerone.com/vulnscan/screenshot/gowitness.db
Error: unknown shorthand flag: 's' in -s


[*] Execute: /root/Osmedeus/plugins/go/gowitness report generate -n /root/.osmedeus/workspaces/hackerone.com/vulnscan/screenshot/hackerone.com-raw-gowitness.html --destination /root/.osmedeus/workspaces/hackerone.com/vulnscan/screenshot/raw-gowitness/ --db /root/.osmedeus/workspaces/hackerone.com/vulnscan/screenshot/gowitness.db
Error: unknown shorthand flag: 'n' in -n_
```

Have also attached the log, Please let me know if you need anything.

[Errors.log](https://github.com/j3ssie/Osmedeus/files/5396351/Errors.log)

I installed the tool by using the commands mentioned on the github page:

git clone https://github.com/j3ssie/Osmedeus
cd Osmedeus
./install.sh

**Comment by JLP04** [comment]
I am also having this issue with amass.

**Comment by DefenceLogic** [comment]
Solved in #176 .There is also a PR for this error

**Comment by j3ssie** [comment]
The issue is no longer exist due to the repo having been upgraded to [Osmedeus Next Generation v4.0](https://twitter.com/OsmedeusEngine)
